src/enums/roles.ts

```diff
@@ -1,7 +1,8 @@
 export enum Roles {
   COMPLETING_PARTY = 'Completing Party',
-  INCORPORATOR = 'Incorporator',
+  CUSTODIAN = 'Custodian',
   DIRECTOR = 'Director',
-  SUBSCRIBER = 'Subscriber',
```

Comment:
Note: I deleted SUBSCRIBER because it's not used in this UI and it's not in the [party schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/party.json).